### ISSUE TITLE
CRDs Part of Chart 

### ISSUE THREAD

**Issue by abebars** [chkk-io/helm-charts#6]
Can the CRDs become part of the helm chart to make the installation process easier?

**Comment by smuneebahmad** [comment]
Hi @abebars , thank you for filing this request. We agree that including CRDs in the Helm Chart simplifies the installation process, but unfortunately, it complicates the upgrade and deletion processes. We made a conscious decision to exclude CRDs from the Helm Chart because Helm currently does not support upgrading or deleting CRDs. This decision followed extensive community discussion due to the risk of unintentional data loss. Unfortunately, there is no community consensus on handling CRDs and their lifecycle at this time [1]

An alternative approach is to create a separate Helm Chart for the CRDs. If we offer this method, the CRD Helm Chart would need to be installed separately.

Please let us know if you would still like to see CRDs in the Helm Chart. We would be happy to consider this as an optional feature (disabled by default).

[1] https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations

**Comment by abebars** [comment]
> Hi @abebars , thank you for filing this request. We agree that including CRDs in the Helm Chart simplifies the installation process, but unfortunately, it complicates the upgrade and deletion processes. We made a conscious decision to exclude CRDs from the Helm Chart because Helm currently does not support upgrading or deleting CRDs. This decision followed extensive community discussion due to the risk of unintentional data loss. Unfortunately, there is no community consensus on handling CRDs and their lifecycle at this time [1]
> 
> An alternative approach is to create a separate Helm Chart for the CRDs. If we offer this method, the CRD Helm Chart would need to be installed separately.
> 
> Please let us know if you would still like to see CRDs in the Helm Chart. We would be happy to consider this as an optional feature (disabled by default).
> 
> [1] https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations

@smuneebahmad Yeah I think there is a benefit of having the CRDs as optional in the Helm chart to simplify the installation
